src/03-control-flow.js

```diff
@@ -146,7 +207,13 @@ function calculateLetterGrade(grade) {}
  *  animalCounts("pig", 1);
  *  //> "1 pig"
  */
-function animalCounts(animal, numberOfAnimals) {}
+function animalCounts(animal, numberOfAnimals) {
+  if (numberOfAnimals > 1) {
```

Comment:
Once again, beautifully clean code.  If we wanted to write this even shorter could we do so using a template literal?